aries-site/src/layouts/main/Footer.js

```diff
@@ -30,25 +35,29 @@ export const Footer = () => {
       }}
     >
       <Box align="center" gap="medium" direction="row-responsive">
-        <Text size="small">
-          &copy; {year} Hewlett Packard Enterprise Development LP
-        </Text>
+        <Text>&copy; {year} Hewlett Packard Enterprise Development LP</Text>
       </Box>
       <Box direction="row" gap="medium">
         {externalFooterLinks.map((link, index) => (
-          <FooterLink
-            size="small"
+          <Button
             key={index}
             href={link.href}
-            label={link.label}
             target="_blank"
             rel="noreferrer noopener"
-          />
+          >
+            <Box>
+              <Text weight="bold">{link.label}</Text>
```

Comment:
Are we going back to `bold` for these footer links? Do they have any hover states?

Reply:
Per a conversation with Chris last week and based on this design, my understanding is they will always be bold and the hover state is just the cursor becomes a pointer

Design: https://www.figma.com/file/K0PppsSh6aQSszqlwKAekg/hpe-design-system-stickers?node-id=382%3A259

Reply:
Update: We should use the library file (not the sticker sheet for design guidance). I'll be updating this PR.